jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/utils/Modal.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.jdi.light.material.elements.utils;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.material.asserts.utils.ModalAssert;
+import org.openqa.selenium.By;
+
+public class Modal extends UIBaseElement<ModalAssert> {
+
+    @JDIAction("Click on '{name}'")
+    public void clickOn() {
```

Comment:
Better implement HasClick interface from JDI which already has this method

Reply:
Implemented